corehq/apps/locations/fixtures.py

```diff
@@ -27,16 +29,35 @@ def __contains__(self, item):
         return item in self.by_id
 
 
-def should_sync_locations(last_sync, location_db):
+def fixture_last_modified(user):
```

Comment:
Perhaps this doesn't make sense right now but if we add more fixtures to this table then it would be better for this function to get all their last modified dates in one query and return and object.

Should this get cached at all?